helm/vmss-prototype/auto-update.md

```diff
@@ -148,96 +145,129 @@ This cluster has tools that automatically apply OS patches and do reboots as nee
+2021-04-01T21:32:35.083747047Z k8s-agentpool1-33778956-vmss	INFO: ===> Completed in 122.30s: ['az' 'vmss' 'deallocate' '--subscription' '00000000-0000-0000-0000-000000000001' '--resource-group' 'testCluster2' '--name' 'k8s-agentpool1-33778956-vmss' '--instance-ids' '1'] # RC=0
+2021-04-01T21:32:35.083828849Z k8s-agentpool1-33778956-vmss	INFO: ===> Executing command: ['az' 'snapshot' 'create' '--subscription' '00000000-0000-0000-0000-000000000001' '--resource-group' 'testCluster2' '--name' 'snapshot_k8s-agentpool1-33778956-vmss' '--source' '/subscriptions/00000000-0000-0000-0000-000000000001/resourceGroups/testCluster2/providers/Microsoft.Compute/disks/k8s-agentpool1-33778k8s-agentpool1-337789OS__1_33c78c2ff87b4bbcad5516f88fab16c1' '--tags' 'BuiltFrom=k8s-agentpool1-33778956-vmss000001' 'BuiltAt=2021-04-01 21:32:35.083641']
+2021-04-01T21:32:43.910981266Z k8s-agentpool1-33778956-vmss	INFO: ===> Executing command: ['az' 'vmss' 'start' '--subscription' '00000000-0000-0000-0000-000000000001' '--resource-group' 'testCluster2' '--name' 'k8s-agentpool1-33778956-vmss' '--instance-ids' '1' '--no-wait']
+2021-04-01T21:32:47.055532206Z k8s-agentpool1-33778956-vmss	INFO: ===> Executing command: ['kubectl' 'uncordon' 'k8s-agentpool1-33778956-vmss000001']
+2021-04-01T21:32:47.143864731Z k8s-agentpool1-33778956-vmss	INFO: ===> Executing command: ['kubectl' 'annotate' 'node' 'k8s-agentpool1-33778956-vmss000001' 'cluster-autoscaler.kubernetes.io/scale-down-disabled-']
+2021-04-01T21:32:47.233459689Z k8s-agentpool1-33778956-vmss	INFO: Creating sig image version - this can take quite a long time...
+2021-04-01T21:32:47.233523790Z k8s-agentpool1-33778956-vmss	INFO: ===> Executing command: ['az' 'sig' 'image-version' 'create' '--subscription' '00000000-0000-0000-0000-000000000001' '--resource-group' 'testCluster2' '--gallery-name' 'SIG_testCluster2' '--gallery-image-definition' 'kamino-k8s-agentpool1-33778956-vmss-prototype' '--gallery-image-version' '2021.04.01' '--replica-count' '3' '--os-snapshot' 'snapshot_k8s-agentpool1-33778956-vmss' '--tags' 'BuiltFrom=k8s-agentpool1-33778956-vmss000001' 'BuiltAt=2021-04-01 21:32:35.083641' '--storage-account-type' 'Standard_ZRS']
+2021-04-01T21:43:34.575631298Z k8s-agentpool2-33778956-vmss	INFO: ===> Completed in 664.61s: ['az' 'sig' 'image-version' 'create' '--subscription' '00000000-0000-0000-0000-000000000001' '--resource-group' 'testCluster2' '--gallery-name' 'SIG_testCluster2' '--gallery-image-definition' 'kamino-k8s-agentpool2-33778956-vmss-prototype' '--gallery-image-version' '2021.04.01' '--replica-count' '3' '--os-snapshot' 'snapshot_k8s-agentpool2-33778956-vmss' '--tags' 'BuiltFrom=k8s-agentpool2-33778956-vmss000000' 'BuiltAt=2021-04-01 21:31:23.482360' '--storage-account-type' 'Standard_ZRS'] # RC=0

```

Comment:
Note that this is getting the new SIG image copy performance that has not yet GA'ed
This is why it is so much faster than before.  Just over 11 minutes to copy/create the SIG image.  The prior run was 7,000 and 12,000 seconds (vs these two at 664 seconds).  That an order of magnitude improvement as shown this test run.